composer.json

```diff
@@ -4,11 +4,8 @@
     "description": "Shipping data export environment for Sylius platform applications",
     "license": "MIT",
     "require": {
-        "php": "^7.3",
-        "sylius/sylius": "~1.8.0 || ~1.9.0"
-    },
-    "conflict": {
-        "api-platform/core": "^2.6"
+        "php": "^7.4 || ^8.0",
+        "sylius/sylius": "^1.10"
```

Comment:
https://github.com/BitBagCommerce/SyliusBonusPointsPlugin/pull/30#pullrequestreview-716851050

Reply:
if this should be >= 1.10 then probably changes are needed here:
https://github.com/BitBagCommerce/SyliusShippingExportPlugin/blob/master/.github/workflows/build.yml#L26